packages/espressocash_api/lib/src/dto/withdraw.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:espressocash_api/espressocash_api.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'withdraw.freezed.dart';
+part 'withdraw.g.dart';
+
+@freezed
+class WithdrawPaymentRequestDto with _$WithdrawPaymentRequestDto {
+  const factory WithdrawPaymentRequestDto({
+    required String senderAccount,
+    required String receiverAccount,
+    required int amount,
+    required int feePercentage,
```

Comment:
`feePercentage` shouldn't be set by this client, it's up for the backend to decide.